main.cpp

```diff
@@ -830,9 +852,13 @@ int main(int argc, char *argv[])
         config.latency_period_steps = std::stoi(opt.latency_period->answer);
 
     // mortality
-    config.use_mortality = false;
-    config.first_mortality_year = 1;  // starts at 1 (same as the opt)
-    config.mortality_rate = 0.0;
+    if (flg.mortality->answer) {
+        config.use_mortality = true;
+        config.mortality_time_lag = std::stoi(opt.first_year_to_die->answer);  // starts at 1 (same as the opt)
+        config.mortality_rate = std::stod(opt.infected_to_dead_rate->answer);
+        config.mortality_frequency = opt.mortality_frequency->answer ? opt.mortality_frequency->answer : "";
```

Comment:
Given the G_option_requires_all above, isn't `opt.mortality_frequency` always set in this case?